tests/unit/test_acoustic_service.py

```diff
@@ -63,18 +63,9 @@ def test_ctms_to_acoustic(
     main_acoustic_fields,
 ):
     contact_list = [example_contact, maximal_contact, minimal_contact]
-    example_contact_expected = [
-        20 + len(main_acoustic_fields),
-        len(example_contact.newsletters) - 2,
-    ]
-    maximal_contact_expected = [
-        20 + len(main_acoustic_fields),
-        len(maximal_contact.newsletters) - 2,
-    ]
-    minimal_contact_expected = [
-        18 + len(main_acoustic_fields),
-        len(minimal_contact.newsletters) - 2,
-    ]
+    example_contact_expected = [52, len(example_contact.newsletters) - 2]
+    maximal_contact_expected = [52, len(maximal_contact.newsletters) - 2]
+    minimal_contact_expected = [33, len(minimal_contact.newsletters) - 2]
```

Comment:
These assertions always struck me as a bit magic. Any way we could rework this so that things are less flaky and/or more obvious?

Reply:
In the end, the PR does not modify these tests